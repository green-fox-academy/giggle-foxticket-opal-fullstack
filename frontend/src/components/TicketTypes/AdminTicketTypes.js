import React from 'react';
import AdminTicket from './AdminTicket';
import './AdminTicketTypes.style.sass'

const fakeData = [
  {
    id: 1,
    title: 'Ticket-Types A',
    description: 'asdf,ffff,f,f,f,f,f,fffffdrdbdb kjbk ',
    iconName:'faAd'
  },
  {
    id: 2,
    title: 'Ticket-Types B',
    description: 'asdf,ffff,f,f,f,f,f,fffffdrdbdbsefwa ',
    iconName:'faAdjust'

  },
  {
    id: 3,
    title: 'Ticket-Types C',
    description: 'asdf,ffff,f,f,f,f,f,fffffdrdbdbfdsf ',
    iconName:'faCheck'

  },
  {
    id: 4,
    title: 'Ticket-Types D',
    description: 'asdf,ffff,f,f,f,f,f,fffffdrdbdbwtg ',
    iconName:'faAddressBook'

  },
];

function AdminTicketTypes() {
  return (
    <div>
      <h1 className="main-title">Ticket Types</h1>
      {fakeData.map(i => <AdminTicket key={i.id} {...i} />)}
    </div>
  );
}

export default AdminTicketTypes;
