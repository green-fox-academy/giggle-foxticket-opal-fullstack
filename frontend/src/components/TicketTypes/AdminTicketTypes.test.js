import React from 'react';
import AdminTicketTypes from './AdminTicketTypes';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<AdminTicketTypes />).toJSON();
  expect(tree).toMatchSnapshot();
});
