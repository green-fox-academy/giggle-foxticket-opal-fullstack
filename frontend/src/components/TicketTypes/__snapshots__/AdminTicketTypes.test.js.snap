// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1
    className="main-title"
  >
    Ticket Types
  </h1>
  <div
    className="container"
  >
    <div
      className="icon"
    />
    <div
      className="content"
    >
      <h1>
        Ticket-Types A
      </h1>
      <p>
        asdf,ffff,f,f,f,f,f,fffffdrdbdb kjbk 
      </p>
    </div>
    <div
      className="btns"
    >
      <button
        className="btn btn--danger--solid btn--medium"
      >
        Edit
      </button>
      <button
        className="btn btn--danger--solid--btn btn--medium"
      >
        Delete
      </button>
    </div>
  </div>
  <div
    className="container"
  >
    <div
      className="icon"
    />
    <div
      className="content"
    >
      <h1>
        Ticket-Types B
      </h1>
      <p>
        asdf,ffff,f,f,f,f,f,fffffdrdbdbsefwa 
      </p>
    </div>
    <div
      className="btns"
    >
      <button
        className="btn btn--danger--solid btn--medium"
      >
        Edit
      </button>
      <button
        className="btn btn--danger--solid--btn btn--medium"
      >
        Delete
      </button>
    </div>
  </div>
  <div
    className="container"
  >
    <div
      className="icon"
    />
    <div
      className="content"
    >
      <h1>
        Ticket-Types C
      </h1>
      <p>
        asdf,ffff,f,f,f,f,f,fffffdrdbdbfdsf 
      </p>
    </div>
    <div
      className="btns"
    >
      <button
        className="btn btn--danger--solid btn--medium"
      >
        Edit
      </button>
      <button
        className="btn btn--danger--solid--btn btn--medium"
      >
        Delete
      </button>
    </div>
  </div>
  <div
    className="container"
  >
    <div
      className="icon"
    />
    <div
      className="content"
    >
      <h1>
        Ticket-Types D
      </h1>
      <p>
        asdf,ffff,f,f,f,f,f,fffffdrdbdbwtg 
      </p>
    </div>
    <div
      className="btns"
    >
      <button
        className="btn btn--danger--solid btn--medium"
      >
        Edit
      </button>
      <button
        className="btn btn--danger--solid--btn btn--medium"
      >
        Delete
      </button>
    </div>
  </div>
</div>
`;
